### ISSUE TITLE
Hyper Parameter for Resnet18 Model

### ISSUE THREAD

**Issue by ganguli9082** [EIDOSlab/UNITOPATHO#1]
Hi,

Just curious about the parameters that are used for training the model for polyp type on the 7000um(subsample 224) patches and the grade classification that was done on the 800um patches for both the subsample of 224x224 and no subsample. I started out using the default settings in the train.py but can't seem to get close to the results described in the paper. 

Thank you for your time.
Alex Ganguli

Preprocess: HE
Apply Transformations: Training images= True,
                                      Testing images = False

learning rate: 0.01 
batch size: 256
num workers: 8
decay factor: 0.1
step size: 20

**Comment by carloalbertobarbano** [comment]
Hello, sorry for the delay in the response.

So, the are two main tasks: 
1. Type classification on 7000µm patches -> here we subsample to 224
2. Grade prediction on 800µm patches -> here we do not subsample the input image

In any case the preprocessing is RGB. Can you try replicating task 2 with one of these hyperparameters settings: https://wandb.ai/eidos/UnitoPath-v1/reports/Grade-predictions--VmlldzoxMzY4NzI5 that should get you around 80% BA for grade prediction (you can click on a single run then go to overview to get the full list of arguments)

**Comment by ganguli9082** [comment]
Great!! Thanks for getting back to me. I’ll let you know how it goes, thank
you.

Cheers,

Alex Ganguli
